<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PortalAluno</class>
 <widget class="QMainWindow" name="PortalAluno">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PortalAluno</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>751</width>
      <height>351</height>
     </rect>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Faculdade</string>
     </attribute>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>270</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Avançar</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtFaculdade">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>150</y>
        <width>341</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 23px;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>140</y>
        <width>311</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Nome da Instituição de ensino: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>20</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Seja Bem-vindo!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAbout">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>270</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Sobre o programa</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Aluno</string>
     </attribute>
     <widget class="QLabel" name="lbl_faculdade">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>10</y>
        <width>441</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 25px;
padding-left: 50%;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_avancarAbaAlunos">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>270</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Avançar</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>361</width>
        <height>251</height>
       </rect>
      </property>
      <property name="title">
       <string>Dados do Aluno</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>334</width>
         <height>196</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Nome do Aluno:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_aluno"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>RA:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="sb_ra">
            <property name="prefix">
             <string/>
            </property>
            <property name="minimum">
             <number>10000000</number>
            </property>
            <property name="maximum">
             <number>99999999</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Data de Nascimento: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="data_nascimento"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Sexo:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="combo_sexo">
            <item>
             <property name="text">
              <string>Masculino</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Feminino</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>50</y>
        <width>341</width>
        <height>211</height>
       </rect>
      </property>
      <property name="title">
       <string>Dados do Curso</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>321</width>
         <height>181</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Curso:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_curso"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Grau:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="combo_grauCurso">
            <item>
             <property name="text">
              <string>Bacharelado</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Técnologo</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Licenciatura</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Pós-graduação</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Período:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="combo_periodo">
            <item>
             <property name="text">
              <string>Matutino</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Noturno</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Integral</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Disciplinas no semestre atual:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="sb_disciplinas"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Disciplinas</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>40</y>
        <width>341</width>
        <height>211</height>
       </rect>
      </property>
      <property name="title">
       <string>Disciplina</string>
      </property>
      <widget class="QWidget" name="layoutWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>321</width>
         <height>181</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Nome:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_disciplina"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_15">
          <item>
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Professor</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_professor"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_16">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>N1:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="sb_n1">
              <property name="decimals">
               <number>1</number>
              </property>
              <property name="maximum">
               <double>10.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_13">
            <item>
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>N2</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="sb_n2">
              <property name="decimals">
               <number>1</number>
              </property>
              <property name="maximum">
               <double>10.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_14">
            <item>
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>N3</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="sb_n3">
              <property name="decimals">
               <number>1</number>
              </property>
              <property name="maximum">
               <double>10.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QPushButton" name="btn_avancarAbaDisciplina">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>270</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Avançar</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Grade de notas</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>751</width>
        <height>271</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Aluno</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Curso</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Disciplina</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>N1</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>N2</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>N3</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Média</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="btn_gerarPdf">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>280</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Exportar em PDF</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_home">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>280</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Adicionar mais alunos</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>747</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
